plugins/module_utils/redfish_utils.py

```diff
@@ -624,6 +625,27 @@ def _get_allowable_values(self, action, name, default_values=None):
             allowable_values = default_values
         return allowable_values
 
+    def check_service_availability(self):
+        """
+        Checks if the service is accessible.
+
+        :return: dict containing the status of the service
+        """
+
+        # Get the service root
+        # Override the timeout since the service root is expected to be readily
+        # available.
+        original_timeout = self.timeout
```

Comment:
I'm on the fence about using this "save/restore" method for the timeout instead of passing in a temporary timeout as part of `get_request`.

Reply:
I prefer to pass `timeout` parameter to `get_request()`
if passed timeout is `None`, then use `self.timeout` inside `get_request()`

Reply:
Sounds good; will update that.

Reply:
Done